custom/enikshay/ucr/reports/beneficiary_register_v2.json

```diff
@@ -0,0 +1,751 @@
+                "format": "default",
+                "transform": {},
+                "aggregation": "simple",
+                "field": "site_choice",
+                "calculate_total": false,
+                "type": "field",
+                "display": "Site"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {},
+                "aggregation": "simple",
+                "field": "treatment_phase",
+                "calculate_total": false,
+                "type": "field",
+                "display": "Rx Phase"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {},
+                "aggregation": "simple",
+                "field": "new_retreatment",
+                "calculate_total": false,
+                "type": "field",
+                "display": "New/Retreatment"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {},
+                "aggregation": "simple",
+                "field": "hiv_status",
+                "calculate_total": false,
+                "type": "field",
+                "display": "HIV status"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {},
+                "aggregation": "simple",
+                "field": "dst_status",
+                "calculate_total": false,
+                "type": "field",
+                "display": "DST status"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {
+                    "type": "translation",
+                    "translations": {
+                        "99dots": "99DOTS",
+                        "contact_centre": "Contact Centre",
+                        "treatment_supporter": "Treatment Supporter",
+                        "merm": "MERM",
+                        "field_officer": "Field Officer"
+                    }
+                },
+                "aggregation": "simple",
+                "field": "adherence_tracking_mechanism",
+                "calculate_total": false,
+                "type": "field",
+                "display": "Adherence Mechanism Assigned"
+            },
+            {
+                "expression": {
+                    "type": "evaluator",
+                    "context_variables": {
+                        "a": {
+                            "datatype": "integer",
+                            "type": "property_name",
+                            "property_name": "total_expected_doses_taken"
```

Comment:
I don't think this will work any more,

Reply:
this is how we do the other calculations. i think it just references other columns

Reply:
but this case property won't exist anymore, right? You've removed it from the task, and the datasource.

Reply:
or are you saying that `property_name` actually references the `expression`, below?

Reply:
that is what i am hoping/thinking it will do. otherwise that implies that it fetches every case and does a property name lookup on that case. i assumed it pulled from the column called that because of the error we saw a few months ago, when we noticed that you are required to have a column for each of the context variables in the expression https://github.com/dimagi/commcare-hq/pull/17835

Reply:
Sounds like this can reference other expressions: https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/userreports#expression-columns